Changes

```diff
@@ -33,11 +33,13 @@ SiliconCompiler 0.6.0 (2022-02-11)
 
 **Minor:**
 
-* Schema: Moving lib 'driver' into cells (consistency)
-* Schema: Adding site symmetry to avoid full lef parser.
-* Schema: Make tool version switch a list
-* Schema: Changing 'asic', 'targetlib' to 'asic', 'logiclib' for clarity.
-
+* Schema: Changed lib 'driver' to move into cells (consistency)
+* Schema: Added site symmetry to avoid full lef parser.
+* Schema: Changed tool version switch to a list
+* Schema: Changed 'asic', 'targetlib' to 'asic', 'logiclib' for clarity.
+* Schema: Changed 'eda, 'report' parameter guideline to always use 'metric' as keyword
+* Schema: Added -skip_check option to speed up new target bringup
+* Schema: Added -skip_step option to enable skipping specific steps
```

Comment:
Should we mention the new tool-specific pdk schema parameters here? ('file', 'directory', 'variable')?
```suggestion
* Schema: Added -skip_step option to enable skipping specific steps
* Schema: Added 'pdk', ['file', 'directory', 'variable'] parameters to enable tool-specific PDK setups.
```